src/types/product.ts

```diff
@@ -0,0 +1,8 @@
+export type ProductConfig = {
```

Comment:
Not sure about the naming, as this does not has all product properties. It's more about the information for product card.

```suggestion
export type ProductCardConfig = {
```

Reply:
Agree with you

Reply:
done!